dev-requirements.txt

```diff
@@ -1,82 +1,86 @@
 -i https://pypi.python.org/simple
+-e .
 -e git+https://github.com/nucypher/py-solc.git@391b8da1a6bac5816877197bda25527c6b0b8c15#egg=py-solc
 aafigure==0.6
 alabaster==0.7.12
-ansible==2.8.0a1
+ansible==2.8.0
 apipkg==1.5
 asn1crypto==0.24.0
 atomicwrites==1.3.0
 attrs==19.1.0
-aws-xray-sdk==0.95
+aws-sam-translator==1.11.0
+aws-xray-sdk==2.4.2
 babel==2.6.0
-boto3==1.9.130
+boto3==1.9.156
 boto==2.49.0
-botocore==1.12.130
+botocore==1.12.156
 bumpversion==0.5.3
 certifi==2019.3.9
-cffi==1.12.2
+cffi==1.12.3
+cfn-lint==0.21.3
 chardet==3.0.4
-commonmark==0.8.1
-coverage==5.0a4
+commonmark==0.9.0
+coverage==5.0a5
 cryptography==2.6.1
 decorator==4.4.0
-docker-pycreds==0.4.0
-docker==3.7.2
+docker==4.0.1
 docutils==0.14
-ecdsa==0.13
+ecdsa==0.13.2
 execnet==1.6.0
 future==0.17.1
 greenlet==0.4.15
 idna==2.8
 imagesize==1.1.0
-jinja2==2.10
+jinja2==2.10.1
 jmespath==0.9.4
-jsondiff==1.1.1
-jsonpickle==1.1
+jsondiff==1.1.2
+jsonpatch==1.23
+jsonpickle==1.2
+jsonpointer==2.0
+jsonschema==2.6.0
 markupsafe==1.1.1
-mock==2.0.0
-more-itertools==7.0.0 ; python_version > '2.7'
-moto==1.3.7
+mock==3.0.5
+more-itertools==7.0.0
+moto==1.3.8
 mypy-extensions==0.4.1
-mypy==0.700
+mypy==0.701
 packaging==19.0
-pbr==5.1.3
-pluggy==0.9.0
+pluggy==0.11.0
 py==1.8.0
-pyaml==18.11.0
+pyasn1==0.4.5
 pycparser==2.19
-pycryptodome==3.8.1
-pygments==2.3.1
-pyparsing==2.3.1
-pytest-cov==2.6.1
+pygments==2.4.1
+pyparsing==2.4.0
+pytest-cov==2.7.1
 pytest-forked==1.0.2
-pytest-mock==1.10.3
+pytest-mock==1.10.4
 pytest-mypy==0.3.2
 pytest-twisted==1.10
-pytest-xdist==1.28.0
-pytest==4.4.0
+pytest-xdist==1.27.0
+pytest==3.10.1
```

Comment:
We'll take it for passing tests - but I'm left wondering why this pin is causing Issues

Reply:
see this PR:  https://github.com/nucypher/py-solc/pulls

Reply:
@cygnusv @KPrasch or this one
https://github.com/nucypher/py-solc/pull/6

I believe either of these will work, but if we want to use pytest4  we need to choose the 2nd one.

Reply:
@vepkenez As far as I understand, `pytest` is a dev dependency of pysolc, so it shouldn't affect nucypher.

Reply:
Both of those PRs fix this issue:
```
item = <Function test_compile_empty_folder>

    def pytest_runtest_setup(item):
>       if (item.get_marker('requires_standard_json') is not None and
            not solc_supports_standard_json_interface()):
E           AttributeError: 'Function' object has no attribute 'get_marker'

src/py-solc/tests/conftest.py:86: AttributeError
```

Reply:
Yeah, I understand that, but isn't that a problem only for pysolc tests? It's not nucypher's problem so I don't see why it should affect nucypher's dependencies.

Reply:
If we merge nucypher/py-solc#6  and update our requirements to pull that commit (because we are sourcing Nucypher's fork of py-solc), then we can put back pytest="*".

Reply:
Ok so it appears that that error is actually coming from a py-solc test that is running because pytest is finding it.  I will look into why it is there.